lib/Wikia/src/Service/User/Preferences/PreferenceServiceImpl.php

```diff
@@ -258,11 +258,7 @@ private function applyDefaults( UserPreferences $preferences ) {
 	}
 
 	private function prefIsSaveable( $pref, $value, $valueFromDefaults ) {
```

Comment:
The problem here was that we're doing a comparison against the defaults and returning early based on that, but we already do a check against the default below, but the following condition should cause the pref to be saved.